use-cases/vehicle-routing-capacity/pom.xml

```diff
@@ -106,11 +106,6 @@
       <version>5.2.3</version>
       <scope>runtime</scope>
     </dependency>
-    <dependency>
-      <groupId>org.webjars</groupId>
-      <artifactId>popper.js</artifactId>
-      <version>1.16.0</version>
```

Comment:
Are you entirely sure about this?
JQuery typically brings this, but there is no way to check that it does unless you've actually checked that dropdowns work in the UI.